### PR TITLE
Upgrade Jest to 22.1.x

### PR DESCRIPTION
**Pull request by AhnpGit**
Upgrade Jest to 22.1.x. Jest 22.1.x offers some new small features, no breaking changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ## Changelog (master)
 
+### v5.0.1
+* Fix: Upgrade Jest to 22.1.x
```
Comment (thymikee) [inline CHANGELOG.md]: How about:
Chore: Upgrade to Jest 22.1.x internally

Comment (AhnpGit) [reply CHANGELOG.md]: no problem :)

**Comment by thymikee** [comment]
@AhnpGit cool, thank you!
Btw, are you interested in becoming a project collaborator? I see that you care and already helping a lot. What do you think? 🙂

**Comment by AhnpGit** [comment]
Thank you, I feel honored to hear so. Even though I care and helped with your project but sometimes I still feel I don't contribute too much to be a collaborator. If you don't mind about that, I'd like to do so.

**Comment by thymikee** [comment]
You're doing just enough already, I've sent you an invite. Feel free to accept or discard it, I appreciate your help in any way 🙂.

**Comment by thymikee** [comment]
@AhnpGit btw if you have any questions, feel free can shoot me on [twitter](https://twitter.com/thymikee) or Discord: thymikee#1279

**Comment by AhnpGit** [comment]
sure :)

**Pull request merged**